roles/ceph-rgw-loadbalancer/tasks/pre_requisite.yml

```diff
@@ -41,7 +41,7 @@
   block:
     - name: set_fact rgw_ports
       set_fact:
-        rgw_ports: "{{ rgw_ports | default([]) + [hostvars[item]['rgw_instances']['radosgw_frontend_port']] | unique }}"
+        rgw_ports: "{{ rgw_ports | default([]) | union(hostvars[item].rgw_instances | map(attribute='radosgw_frontend_port') | map('string') | list) }}"
```

Comment:
```suggestion
        rgw_ports: "{{ rgw_ports | default([]) | union(hostvars[item]['rgw_instances'] | map(attribute='radosgw_frontend_port') | map('string') | list) }}"
```

Reply:
I'm not sure I would call it more readable, but alright, done. :slightly_smiling_face: